src/Backend/FeedServer.vala

```diff
@@ -962,12 +966,12 @@ public class FeedReader.FeedServer : GLib.Object {
 		return m_plugin.getUnreadCount();
```

Comment:
I'm not really sure how to convert this one so I left it alone for now. Feedbin currently ignores this function anyway....